src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,50 @@
 package mate.academy;
 
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.dao.BookDao;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
 public class Main {
+    private static final Injector INJECTOR = Injector.getInstance("mate.academy");
+
     public static void main(String[] args) {
+        BookDao bookDao = (BookDao) INJECTOR.getInstance(BookDao.class);
+
+        Optional<Book> foundBook = bookDao.findById(1L);
+        if (foundBook.isPresent()) {
+            System.out.println("Book found: " + foundBook.get());
+        } else {
+            System.out.println("Book not found");
+        }
+        Book book = new Book();
+        book.setId(1L);
+        book.setPrice(new BigDecimal("100.00"));
+        book.setTitle("Alice");
+        bookDao.create(book);
+        System.out.println("Book created: " + book);
+
+        if (foundBook.isPresent()) {
+            Book bookToUpdate = foundBook.get();
+            bookToUpdate.setTitle("Alice in Wonderland");
+            bookToUpdate.setPrice(new BigDecimal("120.00"));
+            bookDao.update(bookToUpdate);
+            System.out.println("Book updated: " + bookToUpdate);
+        }
```

Comment:
This block of code is attempting to update a book that may not exist. Since 'foundBook' was retrieved before 'book' was created, it's possible that 'foundBook' is empty, and this block would be updating a non-existent book. You should ensure the book exists in the database before attempting to update it.